### PR TITLE
DimensionData adding disk configurations to node object

### PR DESCRIPTION
**Pull request by jadunham1**
Adding new class DimensionDataServerDisk
Adding assert to cover that a disk is present

### CODE REVIEW

**Comment by tonybaloney** [comment]
@jadunham1 this def needs tests against both MCP 1 and MCP 2 locations, I don't know if the response formats are the same.

**Comment by jadunham1** [comment]
Disk looks the same from both MCP1/2, added the test.

File: libcloud/common/dimensiondata.py [hunk 1]
```diff
@@ -602,6 +602,41 @@ def __repr__(self):
...
+        :type  sizeGb: ``int``
+
+        :param speed: Speed of the disk (i.e. STANDARD)
+        :type  speed: ``str``
+
+        :param state: State of the disk (i.e. PENDING)
+        :type  state: ``str``
+        """
+        self.id = id
+        self.scsiId = scsiId
```
Comment (tonybaloney) [inline libcloud/common/dimensiondata.py]: these should really be snake case for consistency.

Comment (jadunham1) [inline libcloud/common/dimensiondata.py]: You're right, not sure what I was thinking, will change

**Comment by tonybaloney** [comment]
Fixed conflicts and merged.

**Pull request closed without merging**